merge_reports.js

```diff
@@ -3,27 +3,42 @@
 
 const { config } = require('./config')
 
+/**
+ * @param {Number} errors the number of errors found
+ * @param {Number} warnings the number of warnings found
+ * @param {Number} notices the number of notices found
+ */
+function getCorrectSummary (errors, warnings, notices) {
```

Comment:
What is a "correct" summary?  Versus an "incorrect" summary?

Reply:
Yes, both of you with Antoine are right. I will fix it.